lib/app.go

```diff
@@ -139,7 +135,7 @@ func appState(app *App, pod *pkgPod.Pod) error {
 	}
 
 	app.State = AppStateCreated
-	createdAt := fi.ModTime().UnixNano()
```

Comment:
@lucab Good catch, the change happens before CRI decides on using nano seconds. 
We need to change it back to unix nano.